puppet/zulip/files/nginx/nginx.conf

```diff
@@ -32,15 +32,13 @@ http {
     gzip_disable "msie6";
     gzip_proxied any;
     gzip_types
-      audio/ogg
       application/javascript
       application/json
       application/xml
       application/x-javascript
-      image/png
       image/svg+xml
       text/css
-      test/javascript
+      text/javascript
```

Comment:
(lol. Fortunately we happen to be sending `application/javascript`, although `text/javascript` is now [preferred](https://tools.ietf.org/html/draft-ietf-dispatch-javascript-mjs).)

Reply:
I guess that's consistent with the fact that we did test it was indeed being compressed via browser tools whenever we edited this in the past :)